File: Component/BusiLib/src/main/java-person/com/component/person/view/IPhotoWallView.java
package com.component.person.view;

import com.component.person.model.PhotoModel;

import java.util.List;

public interface IPhotoWallView {

    void addPhoto(List<PhotoModel> list, boolean clear, int totalNum);

    void insertPhoto(PhotoModel photoModel);

    void deletePhoto(PhotoModel photoModel,boolean numchange);

    void updatePhoto(PhotoModel imageItem);

    void loadDataFailed();
}

File: Component/BusiLib/src/main/java-person/com/component/person/photo/view/IPhotoWallView.kt
package com.component.person.photo.view

import com.component.person.photo.model.PhotoModel

interface IPhotoWallView {

    fun addPhoto(offset: Int, list: List<PhotoModel>?, clear: Boolean, totalNum: Int)

    fun insertPhoto(photoModel: PhotoModel)

    fun deletePhoto(photoModel: PhotoModel, numchange: Boolean)

    fun updatePhoto(imageItem: PhotoModel)

    fun loadDataFailed()
}
